cluster/manifests/prometheus/vpa.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: "poc.autoscaling.k8s.io/v1alpha1"
+kind: VerticalPodAutoscaler
+metadata:
+  name: prometheus
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      application: prometheus
```

Comment:
not prometheus

Reply:
Changed it to use the `pdb-controller` which is not super critical.